asctb-api/src/models/graph.model.ts

```diff
@@ -44,19 +45,38 @@ export class GNode {
     this.comparatorId = '';
     this.comparatorName = '';
     this.name = name;
-    this.metadata = new Metadata(name, ontologyId, label);
+    this.metadata = new Metadata(name, ontologyId, label, bType);
   }
 }
 
 export class Metadata {
-  ontologyId: string;
+  ontologyTypeId: string;
+  ontologyType: string;
   label: string;
   name: string;
+  ontologyId: string;
+  bmType?: string;
 
-  constructor(name: string, ontologyId: string, label: string) {
-    this.name = name;
+  constructor(name: string, ontologyId: string, label: string, bmType?: string) {
     this.ontologyId = ontologyId;
+    if (ontologyId.toLowerCase().startsWith('fma')) {
```

Comment:
Why are you doing this? Ids at this point should already be in the form `${ontologyType}:${ontologyId}`...

Reply:
I was still in the intuition that ID's are not fixed. I will do the necessary changes and will commit again

Reply:
Ok, IDS should be fixed now.